redaxo/src/addons/users/pages/users.php

```diff
@@ -281,6 +287,8 @@
 if ('' != $FUNC_ADD || $user_id > 0) {
     $SHOW = false;
 
+    $self = false;
```

Comment:
wird $self auch irgendwo `true` oder geändert?

was bedeutet `$self = true/false`?

Reply:
Ja, wird in tieferer Ebene gesetzt. Brauchte hier in äußerer Ebene neu einen Default-Wert.
$self bedeutet, ob man sich selbst bearbeitet, also den eigenen User.
Ich pack einen Kommentar dran ;)